webpack/assets/javascripts/foreman_tools.js

```diff
@@ -80,3 +81,22 @@ export function initTypeAheadSelect(input) {
     width: '400px'
   });
 }
+
+// send get forms through turbolinks
+export function tableUpdated(element) {
+  let uri = new URI(window.location.href);
+  let search;
+
+  if (element !== undefined) {
```

Comment:
thanks! I wrote it like that as I know https://github.com/theforeman/foreman/pull/4550/files#diff-8bb38b9ee188f2de0368a1fcad1ede41R87 (once it will be merged) will probably use this code path too..